packages/pds/package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "private": true,
```

Comment:
ah we actually do publish this one for the pds distribution 

https://github.com/bluesky-social/pds